tools/sil-func-extractor/SILFunctionExtractor.cpp

```diff
@@ -249,6 +254,8 @@ int main(int argc, char **argv) {
   Invocation.getLangOptions().DisableAvailabilityChecking = true;
   Invocation.getLangOptions().EnableAccessControl = false;
   Invocation.getLangOptions().EnableObjCAttrRequiresFoundation = false;
+  Invocation.getLangOptions().EnableObjCInterop =
+    Invocation.getLangOptions().Target.isOSDarwin() || EnableObjCInterop;
```

Comment:
`EnableObjCInterop` should take precedent over `Target.isOSDarwin()`.  That is, I should be able to disable ObjC interop even on Darwin.

Reply:
Is there any way to check if `EnableObjCInterop` was defined or not as a command-line argument? I had a look in LLVM docs, and looks like the only options for `bool` CL arguments are `true` or `false` (with `false` as the default) with no way to determine if it was explicitly passed or not.

Reply:
Got it, `llvm::cl::boolOrDefault` does the job.